src/mainboard/pcengines/apu2/mainboard.c

```diff
@@ -250,6 +250,34 @@ static void mainboard_enable(struct device *dev)
 
 static void mainboard_final(void *chip_info)
 {
+	//
+	// AGESA set bit 0, EccExclEn, in register D18F5x240 as 1.
+	// Range was incorrectly enabled to cover all memory
+	// for cases without UMA (no integrated graphics).
+	//
+	// In order to clear EccExclEn DRAM scrubber needs to be
+	// disabled temporarily by setting D18F3x88[DisDramScrub]
+	//
+	u32 val;
+	device_t D18F3 = dev_find_device(0x1022, 0x1583, NULL);
+	device_t D18F5 = dev_find_device(0x1022, 0x1585, NULL);
```

Comment:
@krystian-hebel IMO variables names are not very descriptive. Do we really don't know what this device is?

Reply:
`Host bridge: Advanced Micro Devices, Inc. [AMD] Device 1583`, which isn't any more descriptive in my opinion.

Reply:
@pietrushnic I also have not found a name which could satisfy You. Looked in BKDG and other sources, but found only that these are all memory controller/northbridge devices

Reply:
@miczyg1 @krystian-hebel it looks fine to me, let's send that upstream and see what will happen.